apispec/api.spec.json

```diff
@@ -0,0 +1,99 @@
+{
+    "openapi": "3.0.0",
+    "servers": [
+        {
+            "description": "Local entitlements API",
+            "url": "/api/entitlements/v1/"
+        },
+        {
+            "description": "CI entitlements API",
+            "url": "https://ci.cloud.paas.psi.redhat.com/api/entitlements/v1/"
+        },
+        {
+            "description": "QA entitlements API",
+            "url": "https://qa.cloud.paas.psi.redhat.com/api/entitlements/v1/"
+        },
+        {
+            "description": "PROD entitlements API",
+            "url": "https://cloud.redhat.com/api/entitlements/v1/"
+        }
+    ],
+    "info": {
+        "description": "Service for determining subscriptions in cloud management\n[cloud.redhat.com](http://cloud.redhat.com/api/v1/entitlements)\n",
+        "version": "1.0.0",
+        "title": "Entitlements",
+        "contact": {
+            "email": "ihands@redhat.com"
+        },
+        "license": {
+            "name": "Apache 2.0",
+            "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+        }
+    },
+    "tags": [
+        {
+            "name": "entitlements",
+            "description": "querying your entitlements"
+        },
+        {
+            "name": "api",
+            "description": "RESTful service"
+        },
+        {
+            "name": "permissions",
+            "description": "Operations about user"
+        }
+    ],
+    "paths": {
+        "/services": {
+            "get": {
+                "tags": [
+                    "services"
+                ],
+                "summary": "get a list of services a user is entitled to",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Service"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Response not found"
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "schemas": {
+            "ServiceDetails": {
+                "type": "object",
+                "properties": {
+                    "is_entitled": {
+                        "type": "boolean",
+                        "default": false
+                    }
+                }
+            },
+            "Service": {
+                "type": "object",
+                "additionalProperties": {
+                    "$ref": "#/components/schemas/ServiceDetails"
+                },
+                "example": {
+                    "advisor": {
```

Comment:
shouldn't these examples be something like
```
"insights": {
    "is_entitled": true
},
"smart_management" {
    "is_entitled": false
}
```

Since you are entitled to bundles, not apps?